example/jest.config.js

```diff
@@ -1,10 +1,5 @@
 module.exports = {
   preset: "jest-preset-angular",
-  snapshotSerializers: [
```

Comment:
with ctor params transformer, example app doesn't need these anymore ?

Reply:
We added all three serializers as default preset serializers at some point, so the declaration is redundant.